### PR TITLE
Added ticket maturity to tx page

### PR DESCRIPTION
**Pull request by RogueElement**
Addresses #154

### CODE REVIEW

**Comment by raedah** [comment]
Works well, I like. Saw a few things...
Stakebase output on the vote page is missing.
In addition to tickets, mature applies to votes and coinbase txs as well so should be added to those too.

File: views/tx.tmpl [hunk 1]
```diff
@@ -38,6 +38,22 @@
                         {{.Type}}
                     </td>
                 </tr>
+                {{if eq .Type "Ticket"}}
+                <tr>
+                    <td class="text-right pr-2 h1rem p03rem0">Mature</td>
```
Comment (raedah) [inline views/tx.tmpl]: Capitalize to match rest of template.

File: views/tx.tmpl [hunk 2]
```diff
@@ -38,6 +38,22 @@
...
+                    <td>
+                        {{.TicketMature}}
+                    </td>
+                </tr>
+                {{end}}
+                {{if eq .Type "Vote"}}
+                <tr>
+                    <td class="text-right pr-2 h1rem p03rem0">Funds locked?</td>
```
Comment (raedah) [inline views/tx.tmpl]: LOCKED

File: views/tx.tmpl [hunk 3]
```diff
@@ -38,6 +38,22 @@
                         {{.Type}}
                     </td>
                 </tr>
+                {{if .Mature}}
+                <tr>
+                    <td class="text-right pr-2 h1rem p03rem0">Mature</td>
```
Comment (raedah) [inline views/tx.tmpl]: caps like the rest of the labels

**Comment by chappjc** [comment]
This still has the WIP label.  Is it ready?

**Comment by raedah** [comment]
Yeah its ready.

**Comment by chappjc** [comment]
LGTM, runs fine, but it would be nice if we could limit the length of `OP_RETURN` data with ellipses because this is data with no limit on size.

**Comment by chappjc** [comment]
You may also want to rebase and dep ensure.

**Pull request merged**